karapace/schema_registry.py

```diff
@@ -253,6 +278,8 @@ def subject_version_get(self, subject: Subject, version: Version, *, include_del
             "id": schema_id,
             "schema": schema.schema_str,
         }
+        if schema.references is not None:
+            ret["references"] = schema.references
```

Comment:
```suggestion
            ret["references"] = [reference.to_dict() for reference in schema.references]
```
This is a consequence of the JSON-tightening in #570 ... is this the right way to do this?

Reply:
I already applied these fixes locally but integration tests failed... The latest main fixes break part of protobuf logic... probably because JsonObject or something... will debug it ASAP.